.github/workflows/release-and-publish.yml

```diff
@@ -0,0 +1,69 @@
+# This is a workflow to create github release for a tag and publish the package
+# to npm repo
+
+name: release-and-publish
+
+# Controls when the action will run. Triggers the workflow on push of tag
+# request events but only tags matching the configured regex
+on:
+  push:
+    tags:
+      # patterns to match for tag creation. Here all tags similar to v1.0 or
+      # v1.2.0 will trigger this action
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
+
+# A workflow run is made up of one or more jobs that can run sequentially or in
+# parallel
+jobs:
+  # This workflow contains a single job called "build"
+  create_release:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the
+    # job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access
+    # it
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    # Runs a single command using the runners shell
+    - name: Create release for tag
+      uses: actions/create-release@latest
+      env:
+        # this is provided by github, you don't need to do anything here
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ github.ref }}
+        release_name: Release ${{ github.ref }}
+        body: |
+          Release triggered by actor ${{ github.actor }}
+        draft: false
+        prerelease: false
```

Comment:
Is it possible for us to configure this ? For example when creating a release via GitHub release page, I often choose pre-release for alpha beta or rc tag

Reply:
```suggestion
        prerelease: false
```
This can be

Reply:
These can be configured as I have separate tasks for publishing to `latest` or `next`. So I can keep them as true for `next` release. But which among those fields would like to be `true` or `false` and for which workflow?

Reply:
so far we don't use `draft` so I can safely say we can set `draft` to be always `false`. For `prerelease`, the value should be `false` for official release and `true` for `alpha`, `beta` and `rc`

Reply:
Ok. Updated the code with the same rules.